.coveragerc

```diff
@@ -100,6 +100,10 @@ omit =
     homeassistant/components/brottsplatskartan/sensor.py
     homeassistant/components/browser/*
     homeassistant/components/brunt/cover.py
+    homeassistant/components/bsblan/__init__.py
+    homeassistant/components/bsblan/climate.py
+    homeassistant/components/bsblan/config_flow.py
```

Comment:
Thanks for the review. I see I got some work to do. I'll have a look for the tests in the scaffolding scripts.

Reply:
How can I test the config flow? Couldn't find it in the docs so far

Reply:
Did you run the scaffold script?
```sh
python -m script.scaffold --integration bsblan config_flow
```

It will create a test module template for the config flow for you.

Then use `pytest` to run the tests:
https://docs.pytest.org/en/latest/

```sh
pytest --durations=10 --cov-report term-missing --cov=homeassistant.components.bsblan tests/components/bsblan/
```